include/vicon/udp.h

```diff
@@ -392,8 +357,8 @@ class UDPClient
 
         // ...and, if not, create one
         if (pos == m_ObjectData.cend()) {
-            LOGI << "Vicon: Found new object: " << data.objectName;
-            pos = m_ObjectData.emplace(data.objectName, ObjectDataType { data.objectName }).first;
+            LOGI << "Vicon: Found new object: " << data.objectName.data();
+            pos = m_ObjectData.emplace(data.objectName, ObjectDataType { data.objectName.data() }).first;
```

Comment:
I suppose we could add a ``getName()`` convenience function that returns a ``const char *`` just to make it more explicit